code/modules/admin/ass_day.dm

```diff
@@ -82,6 +82,20 @@ var/global/it_is_ass_day = ASS_JAM //set the BUILD_TIME_DAY in __build.dm to 13
 				Would you like to contribute? Easy Peasy! INSERT LINK TO GITHUB HERE! Warcrimes will surely write a guide to joining the Ass Jam, and totally won't forget to do this before april 13th. <br>
```

Comment:
insert link to github here

Reply:
oh haha i absolutely did forget that didn't i - but lo and behold i will do it eventually!